migration/v0.43/apply/30-gatekeeper-crds.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "installing gatekeeper crds in service cluster"
+      helmfile_do sc -l name=gatekeeper-templates apply --include-transitive-needs
+    fi
+    if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+      log_info "installing gatekeeper crds in workload cluster"
+      helmfile_do wc -l name=gatekeeper-templates apply --include-transitive-needs
+    fi
```

Comment:
Do we need to have this in it's own migration script? Does the helmfile not make sure that we install templates before constraints? (I assume that the problem this is trying to avoid is installing constraints before the templates).
But if we need this, then I wonder if skipping `app!=gatekeeper` is correct in `80-apply.sh`. Since that would also skip installing any changes to the gatekeeper controller and the constraints. Could we either add a full install of gatekeeper (including the app and constraints) in this migration script or have the `80-apply.sh` only skip the templates?